packages/web/src/pages/chat-page/components/ChatMessageListItem.tsx

```diff
@@ -42,6 +44,13 @@ export const ChatMessageListItem = (props: ChatMessageListItemProps) => {
   const senderUserId = decodeHashId(message.sender_user_id)
   const userId = useSelector(accountSelectors.getUserId)
   const isAuthor = userId === senderUserId
+  const reactionUsers = useProxySelector(
```

Comment:
Do we know that the users will be in the cache?

Reply:
yep! I cache all participants in a chat before the chat is in the store